### DISCUSSION TITLE
Melhor fonte para efeito "write" em svg

### DISCUSSION THREAD

**Discussion by kivervinicius** [frontendbr/forum#846] [Perguntas e Respostas]
Gostaria de fazer o efeito de "escrever" em svg usando dash-array o problema é que todas fontes que uso não fica tão legal quanto este https://codepen.io/cameralibre/pen/NpPjYe?q=svg%20write&order=popularity&depth=everything&show_forks=false, como será que consigo chegar nesse efeito, preciso vetorizar todo texto usando um path somente será?

**Comment by willianjusten** [comment]
Olá @kivervinicius , eu tenho [um post explicando sobre como fazer isso](https://willianjusten.com.br/efeito-de-desenhar-com-svg/).

Mas como você notou, sim, você precisa que o elemento seja todo em PATH, por que só assim o efeito vai funcionar corretamente.

Sobre como converter, eu também tenho [um outro post explicando](https://willianjusten.com.br/criando-e-exportando-svg-para-web/) é a dica número 2, basicamente você vai converter para outline, que vai transformar tudo em PATH.

Qualquer coisa, sobre um codepen aí, que vamos ajudando o/

**Comment by kivervinicius** [comment]
Obrigado @willianjusten vou olhar, o efeito eu consigo fazer o maior problema está sendo fazer o path no corel draw rs
